kratos/linear_solvers/power_iteration_eigenvalue_solver.h

```diff
@@ -47,9 +53,126 @@ namespace Kratos
 ///@}
 ///@name Kratos Classes
 ///@{
+    
+/// Utility to initialize a random vector
+/**
+ * Defines several utility functions
+ */
+template<class TDataType>
+class RandomInitializeUtil
```

Comment:
I would move this to a separate file as `VectorRandomInitializer`

Reply:
Ok